CHANGELOG.md

```diff
@@ -2,6 +2,8 @@ Important changes to data-models, configuration and migrations between each
 AppEngine version, listed here to ease deployment and troubleshooting.
 
 ## Next Release (replace with git tag when deployed)
+
+## `20191218t150816-all`
```

Comment:
Could you also add the note about `package:appengine` upgrade?

Reply:
too late.. I already hit deploy :/